File: src/main/java/top/zbeboy/isy/web/vo/internship/release/InternshipReleaseUpdateVo.java
package top.zbeboy.isy.web.vo.internship.release;

import lombok.Data;

import javax.validation.constraints.Min;
import javax.validation.constraints.NotNull;
import javax.validation.constraints.Size;

/**
 * Created by zbeboy on 2016/11/18.
 */
@Data
public class InternshipReleaseUpdateVo {
    @NotNull
    @Size(max = 64)
    private String internshipReleaseId;
    @NotNull
    @Size(max = 100)
    private String releaseTitle;
    @NotNull
    private String teacherDistributionTime;
    @NotNull
    private String time;
    private Byte internshipReleaseIsDel;
    private String files;
}

File: src/main/java/top/zbeboy/isy/web/vo/internship/release/InternshipReleaseUpdateVo.kt
package top.zbeboy.isy.web.vo.internship.release

import javax.validation.constraints.NotNull
import javax.validation.constraints.Size

/**
 * Created by zbeboy 2017-12-18 .
 **/
open class InternshipReleaseUpdateVo {
    @NotNull
    @Size(max = 64)
    var internshipReleaseId: String? = null
    @NotNull
    @Size(max = 100)
    var releaseTitle: String? = null
    @NotNull
    var teacherDistributionTime: String? = null
    @NotNull
    var time: String? = null
    var internshipReleaseIsDel: Byte? = null
    var files: String? = null
}